pkg/appfile/appfile.go

```diff
@@ -682,8 +684,6 @@ func generateTerraformConfigurationWorkload(wl *Workload, ns string) (*unstructu
 	switch wl.FullTemplate.Terraform.Type {
 	case "hcl":
 		configuration.Spec.HCL = wl.FullTemplate.Terraform.Configuration
-	case "json":
-		configuration.Spec.JSON = wl.FullTemplate.Terraform.Configuration
```

Comment:
will it cause incompatible here?

Reply:
[JSON support was deprecated in v0.3.1, and removed in v0.4.6 in Terraform Controller](https://github.com/oam-dev/terraform-controller#supported-terraform-configuration) for these reasons:
- JSON is not friendly for humans who prepare HCL in Terraform Controller or ComponentDefinition in KubeVela: nearly all Terraform modules are written in HCL.
- No user requirements: No feedback from a single user was received.

Reply:
ok